pkg/sysinfo/sysinfo_linux.go

```diff
@@ -33,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		if !sysInfo.OomKillDisable && !quiet {
 			logrus.Warnf("Your kernel does not support oom control.")
 		}
+		_, err = ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.swappiness"))
```

Comment:
@cpuguy83 , that does not solve bail out problem .. I am removing the check itself..